internal/pkg/describe/lb_web_service.go

```diff
@@ -157,9 +161,16 @@ func (d *WebServiceDescriber) Describe() (HumanJSONStringer, error) {
 			return nil, fmt.Errorf("retrieve environment variables: %w", err)
 		}
 		envVars = append(envVars, flattenEnvVars(env, webSvcEnvVars)...)
+		webSvcSecrets, err := d.svcDescriber[env].Secrets()
+		if err != nil {
+			return nil, fmt.Errorf("retrieve secrets: %w", err)
+		}
+		secrets = append(secrets, flattenSecrets(env, webSvcSecrets)...)
```

Comment:
we should think about how we can remove this duplication across workload types the same code seems to be repeated for both LB Web Services and Backend Services

Reply:
And for jobs, eventually, too. Can we leave this refactor for next time?